packages/tasit-action/src/TasitAction.test.js

```diff
@@ -274,22 +278,26 @@ describe("Contract", () => {
       }).to.throw();
     });
 
-    it("should listen to an event", async () => {
+    // Note: Non-Deterministic test (It's failing sometimes).
+    // Failed on CI - Skiping for now
```

Comment:
Skiping -> Skipping

Reply:
Resolved on https://github.com/tasitlabs/TasitSDK/pull/102

Reply:
Okay great - resolving this thread.